input/ignoreWarnings.txt

```diff
@@ -5,7 +5,3 @@ The Implementation Guide contains no examples for this profile
 
 # We believe that this error is caused by infrastructure, not by the content of this IG. See cpb-15's xpath.
 value should not start or finish with whitespace ' not(f:kind/@value='instance')...exists(f:software))'
-
```

Comment:
Eric, Tony already merged this change in here: https://github.com/HL7/fhircast-docs/commit/16fad91ea89b7d59a36dcab174ad62d0cb48ea11.

Can you remove it from your PR / sync with master?